app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
```

Comment:
@ehelms well we have to check for the library instance first, and that will throw a undefined method error if it checks a CVPE for .library_instance
the only other thing I can think of is something like
```
if self.respond_to?(:download_policy)  && self.respond_to?(:library_instance)
```
instead of 

``` 
if self.is_a?(Katello::Repository)
```

but that seems harder to read and less clear